src/classes/STG_InstallScript.cls

```diff
@@ -268,4 +270,18 @@ global without sharing class STG_InstallScript implements InstallHandler {
             upsert orgSettings;
         }
     }
+    
+    /*******************************************************************************************************
+    * @description Populate Default Contact Language Fluency custom setting when upgrading to 1.74
+    * @return void
+    */
+    private static void setDefaultContactLanguageFluency(Version previousVersion) {
```

Comment:
The other methods above that perform certain logic on install/upgrade are global but I have no idea why. It doesn't really make sense to me.

Reply:
@robertbwatson -- I'll see if I can find out why. It might have something to do with being able to have the cci installer stuff call into the class and run specific pieces of setup.